src/common/Executor.cpp

```diff
@@ -998,7 +980,82 @@ void Executor::runKernel(KernelBase* kernel, bool print_kernel_name)
         getCout() << "\t\tSkipping " << tuning_name << " tuning" << endl;
       }
     }
-  } // loop over variants
+  } // iterate over variants
+}
+
+void Executor::runWarmupKernels()
+{
+  if ( run_params.getDisableWarmup() ) {
+    return;
+  } else {
```

Comment:
No need for an else here that causes all of the rest of the code to be indented.

Reply:
Good catch. Fixed.